src/components/ListExport/Popup/Popup.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react'
+import './popup.css'
+
+export default function Popup({ active, handleToHide },props){
+  return(
+    <div className={'popup ' + (active ? 'active' : '')}>
+      <div className='popup-window'>
+      <button onClick={handleToHide} className='btn-close-modal'>
+         <Icon name="/img/x.svg" />
+          
+      </button>
+        <div className="popup-header">
+          <span className="popup-logo">
+              <Icon name="/img/export.svg"/>
+          </span>
+          <span className="popup-title">Exporting</span>
+        </div>
+        <div className="popup-content">
+          <p>
+            You just exported the <b>H-BE SoA</b> canvas a JSON file. 
+            You will be able to find it in your Downloads folder!
+          </p>
+        </div>
+        <div className="popup-footer">
+          <div className="check-no-show-again">
+            <label htmlFor="">Dont't show me again</label>
+            <input type="checkbox"/>
+          </div>
+          <button className="btn-accept">
```

Comment:
Use our custom `<Button />` component here instead, and attach an `onClick` listener to it